src/ert/gui/main_window.py

```diff
@@ -305,6 +324,10 @@ def __add_tools_menu(self) -> None:
         tools_menu.addAction(self.load_results_tool.getAction())
 
     def closeEvent(self, closeEvent: Optional[QCloseEvent]) -> None:
+        for plot_window in self._external_plot_windows:
+            if plot_window:
+                plot_window.close()
```

Comment:
Should maybe be removed from the list as well, but might not matter since since the window is closing anyways :thinking:

Reply:
Honestly don't think we need to unload the list, but can do if you want me to.

Reply:
I think it's fine leaving it as is, since this window is closing as well